test/torchaudio_unittest/datasets/commonvoice_test.py

```diff
@@ -71,3 +123,34 @@ def test_commonvoice_str(self):
     def test_commonvoice_path(self):
         dataset = COMMONVOICE(Path(self.root_dir))
         self._test_commonvoice(dataset)
+
+
+class TestCommonVoiceFR(TempDirMixin, TorchaudioTestCase):
+    backend = 'default'
+    root_dir = None
+    sample_rate = 48000
```

Comment:
Since sample rate is accessible as global variable, there is no need to define another one (duplicated one) on class level.